firstvoices/backend/search/utils/query_builder_utils.py

```diff
@@ -197,6 +232,27 @@ def get_valid_domain(input_domain_str):
         return None
 
 
+def get_valid_starts_with_char(input_str):
+    # Starting alphabet can be a combination of characters as well
+    # taking only first word if multiple words are supplied
+    valid_str = str(input_str).strip().lower().split(" ")[0]
+    return valid_str
```

Comment:
While you're sanitizing input you could probably clean confusables with the `clean_confusables` function on the alphabet model here as well.